lib/IRGen/GenStruct.cpp

```diff
@@ -1512,7 +1523,8 @@ class ClangRecordLowering {
   }
 
   /// Add information to track a value field at the current offset.
-  void addFieldInfo(VarDecl *swiftField, const FixedTypeInfo &fieldType, bool isZeroSized) {
+  void addFieldInfo(VarDecl *swiftField, const FixedTypeInfo &fieldType,
+                    bool isZeroSized) {
```

Comment:
Nit: Was this meant to be part of the patch? It looks to be formatted by mistake. (No problem with including this; I'm just wondering.)